### ISSUE TITLE
sls deploy throws `fs.copyFileSync is not a function`

### ISSUE THREAD

**Issue by cah6** [seek-oss/serverless-haskell#21]
I'm not sure if I'm missing any steps, but I'm unable to run `sls deploy` on either my own project or the example one, with the same error:

```
sls deploy -v
Pulling image from registry: 'fpco/stack-build:lts-9.17'
lts-9.17: Pulling from fpco/stack-build
Digest: sha256:d01545998a8bf2407e3c0e3c2c4b5fa5d00bd9fe449f5d4be388445d6dada741
Status: Image is up to date for fpco/stack-build:lts-9.17

  Type Error ---------------------------------------------

  fs.copyFileSync is not a function
```

I've tried first doing
- `npm install` from `example-project`
- `npm install` from `serverless-plugin`
- `npm install -g fs-extra`
- manually running `stack build --docker` from example-project first

**Comment by koterpillar** [comment]
What is your version of Node.js? We were testing on the latest 9, I think this function was added around 8.5. I'll update the requirements in the documentation.

**Comment by cah6** [comment]
Ah, yeah wow I was on a really old version. Upgrading it to latest worked, thank you!!

**Comment by koterpillar** [comment]
This is fixed in v0.4.1.
